### DISCUSSION TITLE
MHI2 no AM and FM buttons in source

### DISCUSSION THREAD

**Discussion by Jagorius** [Mr-MIBonk/M.I.B._More-Incredible-Bash#128] [General]
Hi
I made a conversion (US->EU) from MHI2_US_POG11_P4151 to MHI2_ER_POG11_K5126. Everything ended well, navigation, android auto, sound (I have bose) works without a problem. The only problem I have is the lack of AM and FM selection buttons from the source tab
![PXL_20230207_174824856](https://user-images.githubusercontent.com/794339/218488777-0971043b-0c55-46a9-ac08-d58b30246358.jpg)

FM works fine, the List button shows a list of all available stations that can be selected. Another problem is that after pressing Tune button nothing happens, there is no window with the option to enter the frequency manually. Has anyone encountered such a problem? How to fix it? Should I try flashing only the tuner?

**Comment by harman-f** [comment]
Did you use the conversion function in M.I.B GEM?
It should take care of all this.

M.I.B. -> Advanced Settings -> Convert to EU train

**Comment by Jagorius** [comment]
Yes, i did the conversions according to: https://mibwiki.one/doc/mhi2-eu-conversion-from-uskrjptwcn-8lv9Zw5DzP

**Comment by harman-f** [comment]
Please share your M.I.B backup and log

**Comment by Jagorius** [comment]
Here are my M.I.B backup and log: https://mega.nz/file/zdhEyaqB#cAogPfaabPKR3npyg1NHZJm3IM1bWq7u8oAHKdVY3xc

**Comment by harman-f** [comment]
I can not see in LOG, that you did run this on M.I.B GEM:
M.I.B. -> Advanced Settings -> 1-Click EU Coding

That will fix coding for EU

**Comment by Jagorius** [comment]
I tried this as well as flashing the tuner itself, but no changes. The AM and FM buttons from the source tab are still invisible and the "Tune" button still doesn't work. Here are my new M.I.B backup and log:
https://mega.nz/file/HEJhVaLD#u8SZnEF0B4NYgisFhdOARgBCM3pGoSxtxO6ZFUmtFo4

**Comment by harman-f** [comment]
Have not seen that before and no idea right away what might cause it.
Looks like some coding issue.
Check manually via OBD if coding is OK now for radio region.

Did you had any issues during US to EU conversion?

Hardware issue?

**Comment by adasxdm** [comment]
Hey, where you able to fix this? I have the same problem… Thanks.

**Reply by sisu1983** [reply]
I have exactly the same problem, has anyone managed to solve it?

**Reply by adasxdm** [reply]
Hey, go to PCM settings and do a Factory reset. If that does not help, in the same page there should be Vehicle handover option, try that. After PCM reset FM tuner should work normal. I did this on few cars and it made FM tuner work.
